src/System.Management.Automation/engine/TypeTable.cs

```diff
@@ -4221,7 +4221,7 @@ public void AddType(TypeData typeData)
 
             // Throw exception if there are any errors
             FormatAndTypeDataHelper.ThrowExceptionOnError("ErrorsUpdatingTypes", errors, RunspaceConfigurationCategory.Types);
-        }
+         }
```

Comment:
This change seems unnecessary. The extra space should be removed.

Reply:
will fix